spec/models/ckey2bibframe_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Ckey2bibframe, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```

Comment:
I added some specs, and I realize that the `marc21_to_xml` and the `marcxml_to_bibframe` tests are not much of real tests, but I am not really sure how to effectively mock and test these methods without spending a lot of time trying to figure it out. Not sure how important it is to fully mock and test these things since they are effectively external resources that are tested by `ld4p-marc21-to-xml` and `marc2bibframe2` respectively.